content/en/blog/2024/sig-cloud-provider-spotlight.md

```diff
@@ -0,0 +1,143 @@
+community has created cloud controllers to address these integration points. Cloud providers can
+create their own controllers either by using the framework maintained by the SIG, or by following
+the API guides defined in the Kubernetes code and documentation. One thing I would like to point out
+is that the SIG cloud provider does not deal with the lifecycle of nodes in a Kubernetes cluster,
+for those types of topics SIG cluster lifecycle and the cluster API project are more appropriate
+venues.
+
+## Important subprojects
+
+**Arujjwal:** There are a lot of subprojects within this SIG, can you highlight some of the most
+important ones and what job they do?
+
+**Michael:** I think the two most important subprojects today are the [cloud provider
+framework](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/README.md#kubernetes-cloud-provider)
+and the [extraction/migration
+project](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/README.md#cloud-provider-extraction-migration). The
+cloud provider framework is a common library to help infrastructure integrators build a cloud
+controller for their infrastructure. This project is most frequently the starting point for new
+people coming to the SIG. The extraction and migration project is the other big subproject and a
+large part of why the framework exists, a little history might help explain further: for a long
+time, Kubernetes has needed to have some integration with the underlying infrastructure, not
+necessarily to add features but to be aware of cloud events like instance termination. The cloud
+provider integrations were built into the Kubernetes code tree, and thus the term "in-tree" was
+created (check out this [article on the topic](https://kaslin.rocks/out-of-tree/) for more
+info). The activity of maintaining provider-specific code in the main Kubernetes source tree was
+considered undesirable by the community. The community’s decision inspired the creation of the
+extraction and migration project to remove the "in-tree" cloud controllers in favour of
+"out-of-tree" components.
+
+
+**Arujjwal:** What makes it a good place to start? Does it have consistent good beginner work? What
+kind?
+
+**Michael:** I feel that the cloud provider framework is a good place to start as it encodes the
+community’s preferred practices for cloud controller managers, and as such will give a newcomer a
+strong understanding of how and what the managers do. Unfortunately, there is not a consistent
+stream of beginner work on this component, this is due in part to the mature nature of the framework
+and that of the individual providers as well. For folks who are interested in getting more involved,
+having some [Go language](https://go.dev/) knowledge is good, and also having an understanding of
+how at least one cloud API (eg. AWS, Azure, GCP) works is also beneficial. In my personal opinion,
```

Comment:
```suggestion
how at least one cloud API (e.g., AWS, Azure, GCP) works is also beneficial. In my personal opinion,
```

Reply:
I would do it as "(eg AWS...)" :D I think here it aligns with the previous change though, and Michael used "(e.g. )" before as well.